ui/ui-frontend/projects/pastis/src/app/core/services/seda.service.ts

```diff
@@ -100,7 +100,7 @@ export class SedaService {
 
     return sedaNode.children.filter(
       (x: SedaData) =>
-        (!fileNodesNames.includes(x.name) && x.cardinality !== CardinalityConstants.ONE_REQUIRED.valueOf()) ||
+        !fileNodesNames.includes(x.name) ||
         (fileNodesNames.includes(x.name) &&
           (x.cardinality === CardinalityConstants.MANY.valueOf() || x.cardinality === CardinalityConstants.MANY_REQUIRED.valueOf())),
```

Comment:
Peut être simplifié en :
```suggestion
        (x.cardinality === CardinalityConstants.MANY.valueOf() || x.cardinality === CardinalityConstants.MANY_REQUIRED.valueOf()),
```

voire :
```suggestion
        ([CardinalityConstants.MANY.valueOf(), CardinalityConstants.MANY_REQUIRED.valueOf()].includes(x.cardinality)),
```


Aussi, le commentaire de la méthode n'est pas/plus raccord.

Reply:
Corrigé.